src/benchmark/config_parser.py

```diff
@@ -361,7 +362,8 @@ def __init__(self, mode, extension, async_request_count, thread_count, stream_co
 
     @staticmethod
     def _mode_is_correct(mode):
-        const_correct_mode = ['sync', 'async', 'ovbenchmark_python_latency', 'ovbenchmark_python_throughput']
+        const_correct_mode = ['sync', 'async', 'ovbenchmark_python_latency', 'ovbenchmark_python_throughput',
+                              'ovbenchmark_cpp_latency', 'ovbenchmark_cpp_throughput']
         if mode.lower() in const_correct_mode:
             return True
         return False
```

Comment:
Let's raise ValueError here, using const_correct_mode variable in error text

Reply:
Added in last commit